content/includes/veteran-navigation.html

```diff
@@ -39,8 +39,7 @@
                   <li><a href="/disability-benefits/conditions/" onClick="reportHeaderNav('Explore Benefits->Disabilty');">Conditions</a></li>
                   <li><a class="login-required" href="/track-claims/" onClick="reportHeaderNav('Explore Benefits->Disabilty');">Track Your Claims and Appeals</a></li>
                   <li><a href="/disability-benefits/claims-appeal/" onClick="reportHeaderNav('Explore Benefits->Disabilty');">Appeals Process</a></li>
-                  <li><a class="usa-button va-button-primary va-external--light" href="https://www.ebenefits.va.gov/ebenefits/about/feature?feature=disability-compensation" onClick="reportHeaderNav('Explore Benefits->Disabilty');">Go to eBenefits to Apply {% include "assets/img/icons/exit-icon-white.svg" %}</a></li>
-                </ul>
+              </ul>
```

Comment:
It looks like the spacing is a bit off here, could `</ul>` be indented as it previously was?